android/runtime/v8/src/native/TypeConverter.cpp

```diff
@@ -663,20 +663,18 @@ jobject TypeConverter::jsValueToJavaObject(v8::Isolate* isolate, JNIEnv *env, v8
 
 			for (int i = 0; i < numKeys; i++) {
```

Comment:
This code looks nearly the same as jsObjectToJavaKrollDict, but in one case we're generating a HashMap, and in the other we're generating a KrollDict. I don't see why that should be the case, and we should consider refactoring this code out to a common method (or maybe we can just call jsObjectToJavaKrollDict here?)